### PR TITLE
Adds redis (in-memory) caching to SqlRepositoryBase

### PR DESCRIPTION
**Pull request by simplicbe**

### CODE REVIEW

File: src/Simplic.Data.Sql/SqlRepositoryBase.cs [hunk 1]
```diff
@@ -60,24 +107,13 @@ public virtual TModel Get(TId id)
...
-            if (UseCache)
-            {
-                obj = cacheService.Get<TModel>(key);
-                if (obj != null)
-                    return obj;
-            }
 
             return sqlService.OpenConnection((connection) =>
             {
                 obj = connection.Query<TModel>($"SELECT * FROM {TableName} WHERE {columnName} = :id",
```
Comment (NexQueek) [inline src/Simplic.Data.Sql/SqlRepositoryBase.cs:114]: Maybe we should use the query builder that dapper provides for better sql queries?

Comment (simplicbe) [reply src/Simplic.Data.Sql/SqlRepositoryBase.cs:114]: @NexQueek is there an existing Sql Anywhere Provider?

Comment (NexQueek) [reply src/Simplic.Data.Sql/SqlRepositoryBase.cs:114]: @simplicbe Its inside of dapper so if we use dapper, what we already do, it should work
Edit: Its provides a better structure of the query

Comment (NexQueek) [reply src/Simplic.Data.Sql/SqlRepositoryBase.cs:114]: https://github.com/Drizin/DapperQueryBuilder

Comment (simplicbe) [reply src/Simplic.Data.Sql/SqlRepositoryBase.cs:114]: Please create an issue for this. We should put it after a review in a separate PR.

**Pull request merged**